aux/vagrant-build/Vagrantfile

```diff
@@ -6,6 +6,8 @@ boxes = [
   {:name => 'wheezy',   :libvirt => 'fm-debian7',    :image_name => /Debian.*7/,      :os_user => 'debian'},
   {:name => 'f19',      :libvirt => 'fm-fedora19',   :image_name => /Fedora.*19/, :pty => true},
   {:name => 'f20',      :libvirt => 'fm-fedora20',   :image_name => /Fedora.*20.*PVHVM/, :pty => true},
+  {:name => 'f21',      :libvirt => 'fm-fedora21',   :image_name => /Fedora.*21.*PVHVM/, :pty => true},
+  {:name => 'f22',      :libvirt => 'fm-fedora22',   :image_name => /Fedora.*22.*PVHVM/, :pty => true},
```

Comment:
Yeah right, actually our Jenkins/Raskspace ceased working from the last week (they deleted old Fedora versions). I will do separate one.